src/openrct2/management/Marketing.cpp

```diff
@@ -46,16 +46,16 @@ int32_t marketing_get_campaign_guest_generation_probability(int32_t campaign)
     switch (campaign)
     {
         case ADVERTISING_CAMPAIGN_PARK_ENTRY_FREE:
-            if (park_get_entrance_fee() < 4)
+            if (park_get_entrance_fee() < MONEY(4, 0))
```

Comment:
Er, that's not what I said.

Reply:
`clang-format` enforces the space though, doesn't it?

Reply:
This was just the `clang-format` patch. Hadn't read your comment at the time.